### ISSUE TITLE
Can I use a custom font in version 3.2?

### ISSUE THREAD

**Issue by apple01150525** [mathjax/MathJax#3075]
I saw in the official document that the current version 3.2 supports several fonts. Can custom fonts be supported currently? Or is there a way that I can replace the font? Because I want to show math formulas to kids, and wish the font cute.

Looking forward to your reply.

**Comment by dpvc** [comment]
No, MathJax 3.2 does **not** support multiple fonts.  That has been added for v4.0, which is currently in beta release, and includes support for 11 fonts.  None of them would likely be termed "cute".  in order to use a font, MathJax needs extensive data about the font metrics, character bounding boxes, and so on, and this data needs to be made available in data files for MathJax.  The tools for making these file will be made available after the v4.0 release, once they are cleaned up and documented.  So after that, you would be able to make the data needed for your own font, but that will not be available for a while.

**Comment by kevcam4891** [comment]
Any update on how the tool is coming along to generate font data files? We'd love to get access to it!

**Comment by dpvc** [comment]
Unfortunately, v4 is still in beta release.  We are in the code cleanup phase (no new features), and updating the documentation.  Once we get the v4 release officially out, work can begin on making the font tools ready for public use.

**Comment by kevcam4891** [comment]
@dpvc Hi! We are really needing a tool like this because we have a customer requiring us to render math in Verdana. We'd be willing to contribute to this project if there's anything we can do to assist financially to prioritize this feature. Let me know if you'd be interested in discussing offline. Thanks!

**Comment by dpvc** [comment]
@kevcam4891, send me email at dpvc at mathjax dot org, and I will be glad to discuss the possibilities with you.
